package.json

```diff
@@ -37,7 +37,7 @@
     "seed:make": "knex seed:make",
     "seed:run": "knex seed:run",
     "seed:run:test": "NODE_ENV=testing knex seed:run",
-    "dev": "tsc && npm run copy && tsc -w & nodemon ./build/src/index.js",
+    "dev": "nodemon",
```

Comment:
I'd been having trouble with the `dev` script not properly re-building and/or re-running when changes are made.

I had a google and found that people tend to use the `exec` command rather than pointing nodemon at the build output directly. This is done in the `nodemon.json` config file I added as part of this PR.

It's been more reliable since, and also doesn't do that weird restart-loop when you first run it and it's still building.